docs/source/model_design/concept_model/double_counting_issues/index.rst

```diff
@@ -31,6 +31,24 @@ Double Counting Issues
    :local:
    :depth: 1
 
-.. todo:: 
+Intervention effects
+--------------------
 
-  This section
\ No newline at end of file
+In designing a Vivarium concept model, attention should be paid to ensure that we will not overestimate the impact of interventions that act on the same (or related) outcomes. For example, we would not want to include the effects of *both* iron and folic acid supplementation (IFA) *and* multiple micronutrient supplementation (MMS) containing IFA in the :ref:`antenatal supplementation intervention <maternal_supplementation_intervention>`; rather, we model the impact of MMS containing IFA *relative to* the impact of IFA. 
+
+Furthermore, two separate interventions may or may not be as valuable as the sum of their parts. For example, cash transfers and small quantity lipid-based nutrient supplementation (SQ-LNS) have been shown to reduce child wasting burden. However, it is possible that cash transfers may reduce child wasting burden in part through the provision of nutrients included in SQ-LNS, which may render the impacts of each intervention when delivered in combination less than the individual impacts of each intervention delivered independently. In such cases, research on intervention impacts in tandem and/or discussion with subject matter experts will be the best source to inform modeling strategies.
+
+Finally, an analysis by [Scott-et-al-2020-double-counting]_ discusses a strategy for modeling multiple interventions that act on the same outcomes in a priority-identifying manner that has been used in multiple nutrition models in the **Intervention expansion pathways** section of the methods.
+
+Morbidity and mortality
+-----------------------
+
+While the GBD cause-hierarchy generally prevents double counting while modeling causes at the same level of the hierarchy, attention should be paid to ensure double counting is not present when modeling causes from varying levels of the hierarchy (for example: modeling both maternal disorders (cause ID 366) *and* maternal hemorrhage (cause ID 367), which is a sub-cause of maternal disorders).
+
+Additionally, double counting issues will be especially salient when modeling GBD impairments, which represent health states due to many different causes in the GBD cause hierarchy. An example of a modeling strategy to avoid such issues is our vivarium model of the :ref:`anemia impairment <2019_anemia_impairment>` (that utilizes our model of :ref:`hemoglobin concentration <2019_hemoglobin_model>`) and our model of :ref:`maternal disorders YLDs <2019_cause_maternal_disorders>`, which were adjusted to omit YLDs associated with anemia due to maternal hemorrhage (See the *Years Lived With Disability* section). Otherwise, we may have double counted these YLDs by observing them both in our anemia model *and* maternal disorders model for the :ref:`IV iron simulation <2019_concept_model_vivarium_iv_iron_maternal_sim>`.
```

Comment:
@SylLutze : Heart Failure could be another good example to link to in this section